.pylintrc

```diff
@@ -195,7 +195,7 @@ indent-string='    '
 [SIMILARITIES]
 
 # Minimum lines number of a similarity.
-min-similarity-lines=4
```

Comment:
@clintonb Doing so results in the following error: "E265 block comment should start with '# '". When I add a space after the pound sign (#), I still get an error about duplicate code.